ais/models.py

```diff
@@ -852,6 +863,10 @@ class AddressSummary(db.Model):
     geocode_type = db.Column(db.Text)
     geocode_x = db.Column(db.Float)
     geocode_y = db.Column(db.Float)
+    # geocode_curb_x = db.Column(db.Float)
+    # geocode_curb_y = db.Column(db.Float)
+    geocode_street_x = db.Column(db.Float)
```

Comment:
if the names are really the issue then i can make them less ambiguous - like geocode_street_x -> geocode_parcel_x_in_street or something like that. i thought we were trying to avoid joining tables on the fly if unnecessary (like the temporary street_code to make tencode - unless the prefered design with street_code in address_summary was motivated by putting street_code in address summary for the utility of users of that table)